io/include/pcl/io/hdl_grabber.h

```diff
@@ -214,6 +230,29 @@ namespace pcl
           double cosVertOffsetCorrection;
       };
 
+      void
+      fireCurrentSweep ();
```

Comment:
WILCO.  The Eclipse Formatter didn't do those, for some reason.  Same with the indentation on the return statement.  Not sure why, though..